Add task to DRF Serializers topic (#2)
mate-academy/py-music-school: README.md

# Music School

**Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)

> before start.

Reviewer: starting.

> Welcome to our Music School.

Reviewer: Welcome to our Music School! Many musicians apply to us every day, and we need you to create a model for it named **Musician**.

A lot of musicians apply to us every day and we need you to create model for it named `Musician`.

This model should have such fields:
* `first_name` — `CharField` with the `max_length` of 64;
* `last_name` — `CharField` with the `max_length` of 64;

> * `instrument` — `CharField` with the `max_length` of 64;

Reviewer 2: I know, that previously I said, that 64 is a good choice, but 63 seems to be more good from the memory prospective.
Sorry for that.

> * `age` - `IntegerField` (we **do not** accept people who are under 14);

Reviewer: тут короткое тире, нужно длинное —

https://monosnap.com/file/3BcJWLayjlkMnubznU6xMd5TVHGIyh

> * `date_of_applying` - `DateField` with `auto_now_add` parameter.

Reviewer: —

> Implement the `__str__` method in this model, so it should return: `"first_name last_name"`.

Reviewer: Implement the `__str__` method in this model to return the string in the `"first_name last_name"` form.

> Also, we need to know whether the musician is adult, so implement the `is_adult` property (adult person is the one who is 21+).

Reviewer: Also, we need to know whether the musician is an adult, so implement the `is_adult` property (an adult person is the one who is 21+). And last but not least, implement the most basic `CRUD` functionality!

> And last but not least, make all CRUD work!

Reviewer 2: ```suggestion
And last but not least, implement the most basic CRUD functionality!
```

> **Please note:** `is_adult` property should be displayed in GET requests.

Reviewer: Please note: the `is_adult` property should be displayed in the `GET` requests.
